### PR TITLE
Feature/footer - update scroll

### PR DESCRIPTION
**Pull request by Far3llProject**

### CODE REVIEW

File: components/NaveBelieve.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <section id="believe" class="flex flex-col items-center md:flex-row nave-believe py-48">
+  <section id="believe" ref="believe" class="flex flex-col items-center md:flex-row nave-believe py-48">
```
Comment (DouglasCalora) [inline components/NaveBelieve.vue:2]: remover ref

File: components/NaveAbout.vue [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <section id="about" class="flex items-center nave-about py-24">
+  <section id="about" ref="about" class="flex items-center nave-about py-24">
```
Comment (DouglasCalora) [inline components/NaveAbout.vue:2]: remover ref

File: components/NaveHeader.vue [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div>
-    <header class="bg-white fixed inset nave-header py-5 shadow-md w-full z-50">
+    <header ref="header" class="bg-white fixed inset nave-header py-5 shadow-md w-full z-50">
```
Comment (DouglasCalora) [inline components/NaveHeader.vue:3]: remover ref

**Review by DouglasCalora — CHANGES_REQUESTED** [review]

**Review by DouglasCalora — APPROVED** [review]

File: components/NaveHeader.vue (hunk 3 above)
Comment (Far3llProject) [reply components/NaveHeader.vue]: ok

**Pull request merged**